ggml/src/ggml.c

```diff
@@ -185,7 +185,7 @@ static void ggml_print_backtrace_symbols(void) {
         fprintf(stderr, "%d: %p %s\n", idx, addr, symbol);
     }
 }
-#elif defined(__linux__)
+#elif defined(__linux__) && defined(GLIBC_BACKTRACE_SUPPORTED)
```

Comment:
Unfortunately this will not work with the `Makefile` build, which many people still depend on. There are other standard headers included before, so `__GLIBC__` should be defined at this point if using glibc. Would this work instead?
```suggestion
#elif defined(__linux__) && defined(__GLIBC__)
```

Reply:
I experienced this issue on Alpine (musllinux) and workaround was using:
```bash
apk add --repository=https://dl-cdn.alpinelinux.org/alpine/v3.16/main/ libexecinfo-dev
```

Reply:
Was not aware of the `__GLIBC__` macro. That is a much cleaner solution.

PR is updated